app/uk/gov/hmrc/agentservicesaccount/controllers/amls/AmlsNewSupervisoryBodyController.scala

```diff
@@ -42,37 +41,47 @@ class AmlsNewSupervisoryBodyController @Inject() (actions: Actions,
 )(implicit appConfig: AppConfig, ec: ExecutionContext) extends FrontendController(cc) with AmlsJourneySupport with I18nSupport {
 
 
-  def showPage: Action[AnyContent] = actions.authActionCheckSuspend.async { implicit request  =>
+  def showPage(cya: Option[Boolean]): Action[AnyContent] = actions.authActionCheckSuspend.async { implicit request  =>
```

Comment:
Does this boolean need to be optional? What do you need three states for in this case? I would assume that cya is either true or false and you could default it to false.
```suggestion
  def showPage(isCya: Boolean = false): Action[AnyContent] = actions.authActionCheckSuspend.async { implicit request  =>
```

Reply:
Yes, good point, thanks.